### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by safafa**
In this pull request I : 

- Created a new module for CRUD functions.
- Implemented the create, remove and clear all completed projects.
- The edit is working on double click on the text description. 
[Live Demo link](https://safafa.github.io/To_Do_list/)

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -1 +1,18 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <title>To do list</title>
+<script defer src="/To_Do_list/bundle.js"></script></head>
+<body>
+    <div id="to_do_list">
+        <ul id="ulTask">
+            <li><h4>Today's to do</h4></li>
+            <li><form id="form"><input type="text" placeholder="Add to your list..." id="task" name="task"><button type="submit" id="addTask"><i class="fab fa-pushed"></i></button></form></li>
```
Comment (rloterh) [inline dist/index.html:13]: - [ ] Your project is apparently **not** fully completed,,seems like you mixed up the branches . The **demo link** is showing current project but the branch `crud` which you propose to merge does not contain most required features implemented. It currently cannot **create**,  **read**,  **update** and **delete**. Kindly ensure to correct this issue

**Project on Branch `crud` looks like this:**
image

**Demo link project looks like this:**
image

- [ ] Demo link project: Your input element is accepting empty fields. Kindly ensure to validate this input to check that the input is not empty before data is stored.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Safa, 

You did extremely great on this project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 

## Good Points 👍  
- No linter errors


## Required Changes ♻️
- [ ] _Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/index.html (hunk 1 above)
Comment (safafa) [reply dist/index.html:13]: @rloterh I ran the project locally using "npm run start" on crud branch and everything is working fine on my side. it's exactly like the demo link.
Form validation is not part of the requirements of the project.

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**